javascript-challenges.test.js

```diff
@@ -14,21 +14,81 @@
 
 // a) Create a test with expect statements using the variables provided.
 
-const secretCodeWord1 = "Lackadaisical"
+// describe("codeWord" , () => {
+//     it("returns a string with a coded message", () => {
+//         expect(codeWord(secretCodeWord1)).toEqual("L4ck4d41s1c4l")
+//         expect(codeWord(secretCodeWord2)).toEqual("G0bbl3dyg00k")
+//         expect(codeWord(secretCodeWord3)).toEqual("3cc3ntr1c")
+//     })
+// })
+
+// const secretCodeWord1 = "Lackadaisical"
 // Expected output: "L4ck4d41s1c4l"
-const secretCodeWord2 = "Gobbledygook"
+// const secretCodeWord2 = "Gobbledygook"
 // Expected output: "G0bbl3dyg00k"
-const secretCodeWord3 = "Eccentric"
+// const secretCodeWord3 = "Eccentric"
 // Expected output: "3cc3ntr1c"
 
 // b) Create the function that makes the test pass.
 
+// const codeWord = (string) => {
+//     let codedMessage = ""
+//     for (let i = 0; i < string.length; i++) {
+//         let char = string[i].toLowerCase()
+//         switch (char) {
+//             case 'a':
+//                 codedMessage += '4';
+//                 break;
+//                 case 'e':
+//                 codedMessage += '3';
+//                 break;
+//                 case 'i':
+//                 codedMessage += '1';
+//                 break;
+//                 case 'o':
+//                     codedMessage += '0';
+//                     break;
+//                     default:
+//                     codedMessage += string[i]
+//         }
+//     }
+// return codedMessage
+// }
+
+// console.log(codeWord(secretCodeWord1))
+// console.log(codeWord(secretCodeWord2))
+// console.log(codeWord(secretCodeWord3))
```

Comment:
Just a reminder that the test will call your function. These three lines should not be a part of your workflow!